src/sentry/nodestore/django/backend.py

```diff
@@ -1,49 +1,57 @@
 from __future__ import absolute_import
 
 import math
+import logging
+import pickle
 
 from django.utils import timezone
 
 from sentry.db.models import create_or_update
 from sentry.nodestore.base import NodeStorage
+from sentry.utils.strings import decompress, compress
 
 from .models import Node
 
+logger = logging.getLogger("sentry")
+
 
 class DjangoNodeStorage(NodeStorage):
     def delete(self, id):
         Node.objects.filter(id=id).delete()
         self._delete_cache_item(id)
 
-    def get(self, id):
-        item_from_cache = self._get_cache_item(id)
-        if item_from_cache:
-            return item_from_cache
+    def _decode(self, value, subkey):
+        if value is None:
+            return None
+
+        try:
+            if value.startswith(b"{"):
```

Comment:
not that it matters, but this now means that you can save `[{}]` to nodestore but you cannot read it (will error in the pickle path).

Reply:
Yeah. Unfortunately pickle has no magic bytes so this is the best I could come up with that still performs well.